netweaver/install_pas.sls

```diff
@@ -9,11 +9,12 @@
 
 {% set product_id = node.product_id|default(netweaver.product_id) %}
 {% set product_id = 'NW_ABAP_CI:'~product_id if 'NW_ABAP_CI' not in product_id else product_id %}
+{% set inifile = '/tmp/pas.inifile'~instance_name~'.params' %}
```

Comment:
same here, and for other instances as well the name for variable `inifile` could be changed

Reply:
Honestly, as we can use the name we really want, I prefer to have the most descriptive name possible. I don't see any value having a generic name.